src/components/contactList/contactList.jsx

```diff
@@ -2,19 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ContactItem from '../contactItem/contactItem';
 
-const ContactList = ({ contacts }) => (
+const ContactList = ({ contacts, onDeleteClick }) => (
   <React.Fragment>
     {(contacts.length === 0) && <p className="mt-4">No results found :(</p>}
     <table className="table mt-4">
       <tbody>
-        {contacts.map(item => <ContactItem key={item.id} name={item.name} phone={item.phone} />)}
+        {contacts.map(item => (
+          <ContactItem
+            id={item.id}
```

Comment:
You got a lot of fields already, I would suggest to pass the whole object as `contact` prop and make destructuring inside ContactItem